### ISSUE TITLE
Unable to import SpatialNavigationDeviceTypeProvider. 

### ISSUE THREAD

**Issue by connor-iversoft** [bamlab/react-tv-space-navigation#87]
**Describe the bug**

Unable to access SpatialNavigationDeviceTypeProvider from standard import. 

**To Reproduce**
```tsx
import { SpatialNavigationDeviceTypeProvider } from 'react-tv-space-navigation';
```

**Expected behavior**
SpatialNavigationDeviceTypeProvider to be imported

**Screenshots**

**Version and OS**
 - Library version: 2.1.0
 - React Native version: 0.72.0 
 - OS web

**Additional context**

As stated in the docs, I attempted to import SpatialNavigationDeviceTypeProvider and wrap our app in it but an error states no export member exists. The example code directly access it via src. When trying to access it direct from DeviceContext folder in dist or src `Module Not Found` errors were returned.

Apologies if this is misunderstanding how to access this provider. Thanks for the assistance!

**Comment by pierpo** [comment]
Oh! Indeed we forgot to export it.
Sorry about this little mistake 🙈 

I'll publish a fix this morning!

**Comment by pierpo** [comment]
Should be fixed with v2.1.1. Thank you for reporting this 🎉

**Comment by connor-iversoft** [comment]
@pierpo Thanks for the quick fix 🙏!
